Week1/README.md

```diff
@@ -4,14 +4,9 @@
 
 These are the topics for week 1:
 
-1. What is an information (system)?
-2. What are entities?
-3. What is a database?
-4. What is the role of a database in an application?
-5. What is Structured Query Language (SQL)?
-6. What are data types (as applied to databases)?
-7. How to use SQL to Create, Read, Update and Delete (CRUD)
-8. What is a database dump?
+1. [Introduction to databases](https://hackyourfuture.github.io/study/#/databases/README)
+2. [Introduction to Structured Query Language (SQL)](https://hackyourfuture.github.io/study/#/databases/SQL)
+3. [Using database dumps](https://hackyourfuture.github.io/study/#/databases/dumps)
```

Comment:
This link also gives 404

A suggestion to check all links by copy-pasting the URLs into a browser.